cfme/rest/__init__.py

```diff
@@ -297,3 +297,22 @@ def _finished():
             rest_api.collections.service_templates.action.delete(*s_tpls)
 
     return s_tpls
+
+
+def automation_requests_data():
+    return [{
+        "uri_parts": {
+            "namespace": "System",
+            "class": "Request",
+            "instance": "InspectME",
+            "message": "create",
+        },
+        "parameters": {
+            "vm_name": "test_rest_{}".format(fauxfactory.gen_alphanumeric()),
+            "vm_memory": 4096,
+            "memory_limit": 16384,
```

Comment:
Looking at this, you sure that this is entirely correct? InspectMe expects some DB object passed, like VM, or Host, this is not a provisioning request. I know this is just moving of data but I haven't spotted this before :)